_posts/en/2021-12-8-throwback-to-nantes-2021-devfest-my-first-conference.md

```diff
@@ -0,0 +1,169 @@
+
+### Quick guide to start a design system
+
+_By Cécile Freyd-Foucault_
+
+When and how should you start a design system?
+
+This is the question Cecile answered, giving us a ton of tips to start. It’s clear she had a lot of experiences about it, whether good or bad. She was able to learn from her mistakes and explain to us how to succeed building our design system.
+
+What seems primary for her is to start a design system for the good reasons and in a team (at least a designer and a developer). You have to explain why its’ necessary and convince the persons in charge while being totally transparent (pros, cons, workload & organization).
+
+Then you’ll have to do an inventory, prioritize, document and engage people around the design system, both internally and externally.
+
+I personally hope to have the opportunity to create a design system in a future project, as this is one of my goals as a developer. When this day will come, I’ll take my notes and I’ll apply this precious advices from Cécile!
+
+### What if you learned how to code to your children
+
+_By Stéphanie Moallic_
+
+I don’t have children, but I was strangely curious about Stéphanie’s talk. Until then, I had no idea how to introduce programming to kids. Stéphanie has a daughter. “Mom, what is your job?” is a question she often has been asked, so decided out to answer her daughter involving her in toys programming!
+
+Stéphanie explain us how to make programming playful, thanks to equipment and technologies linked to each other that allow one to animate toys, robots, etc... Thanks, for instance, to the [micro:bit](https://en.wikipedia.org/wiki/Micro_Bit) card. She even demonstrated it with her own robots, cars, and a Lego connected house!
+
+During her talk, Stéphanie made me feel like a kid and in the end, I would really like to try this out. She will admit it, she’s the one playing most with all these toys!
+
+### React Query, the easy server state for React
+
+_By Olivier Thierry_
+
+As a React developer, I was looking forward to this “quickie”. Olivier showcases us React Query, a server state manager (unlike client state manager’s Redux) developed by Tanner Linsley, well known for his contributions to the ReactJS community ([his GitHub](https://github.com/tannerlinsley) will confirm).
+
+The main features of React Query are front-end/back-end synchronization, REST/GraphQL fetching, the query state management, cache management and cached data update. To enjoy these features, the library give us many hooks (`useQuery`, `useMutation` and `useQueryClient`).
+
+Advanced features are also available, like paginated queries, “See more”, infinite scroll, query canceling, etc...
+
+Olivier had the time to show us main features with source code and tell us about advanced ones in 15 minutes! Hats off to him, because in this short amount of time, he made me want to use it in my next side project...
```

Comment:
```suggestion
Olivier had the time to show us the main features with source code and tell us about advanced ones in 15 minutes! Hats off to him, because in this short amount of time, he made me want to use it in my next side project...
```